Sprint 1/Estimation Documents/MoSCoW.md

```diff
@@ -0,0 +1,30 @@
+#App:
+    **Must:**
+        Have a questionaire to enter data
+        Send data to the server
+        Work on iOS devices
+    **Should:**
+        Sanitise SQL inputs (or otherwise defend against harm)
+        Have a good UI
+        Work on android devices
+        Check for invalid data
+    **Could:**
```

Comment:
Here are some other could ideas:
* Admin interface to access the data
* Web app